### PR TITLE
Devnet 0.6.4 rc 3713

### PR DESCRIPTION
**Pull request by ebmifa**
https://github.com/MystenLabs/sui/pull/3713

### CODE REVIEW

**Review by bmwill — CHANGES_REQUESTED** [review]
Why are you trying to merge this into main?

File: RELEASES.md [hunk 1]
```diff
@@ -3,9 +3,10 @@
 DevNet releases can be found in the `devnet` branch.
 
 ## Current Release
-* 22-07-19: [0.6.1](https://github.com/MystenLabs/sui/releases/tag/devnet-0.6.1)
+* 22-07-26: [0.6.2](https://github.com/MystenLabs/sui/releases/tag/devnet-0.6.2)
```
Comment (Clay-Mysten) [inline RELEASES.md:6]: Consider adding 0.6.3 and 0.6.4 to this list.

**Review by Clay-Mysten — APPROVED** [review]
The Markdown here LGTM with one suggestion in comment.

**Review by huitseeker — CHANGES_REQUESTED** [review]
@ebmifa what's going on?

**Comment by ebmifa** [comment]
@tharbert and I are working on a separate merge, that will include this one. Closing this.

**Pull request closed without merging**